### DISCUSSION TITLE
Getting Error: Path cannot be empty (View: /var/task/resources/views/templates/clean.blade.php)

### DISCUSSION THREAD

**Discussion by JeanHules** [stefanzweifel/sidecar-browsershot#88] [Q&A]
BrowsershotLambda::html(View::make('templates.clean'));

Any ideas why this would be happening?

I am using Laravel 10 on Vapor.

If I can get this working, this package will have saved my life!

**Comment by stefanzweifel** [comment]
@JeanHules You have to pass a string to the `BrowsershotLambda::html()`-method. `View::make()` returns a `Illuminate\View\View`-instance.

Does this work?

```php
$html = View::make('templates.clean')->render();
BrowsershotLambda::html($html);
```
